src/Main.java

```diff
@@ -1,4 +1,44 @@
 public class Main {
     public static void main(String[] args) {
+        //System.out.println("Задача 1");
+        int[] array1 = new int[]{1, 2, 3};
+        float[] array2 = {1.57f, 7.654f, 9.986f};
+        String[] array3 = {"Один", "Два", "Три"};
+
+        System.out.println("\nЗадача 2");
+        for (int i = 0; i < 3; i++) {
+            String separator = (i != array1.length - 1) ? ", " : "\n";
+            System.out.print(array1[i] + separator);
+        }
+        for (int i = 0; i < 3; i++) {
```

Comment:
```suggestion
        for (int i = 0; i < array2.length; i++) {
```
надо использовать длину массива, а не число с потолка